packages/orchestrator-ui-components/src/messages/en-US.json

```diff
@@ -53,6 +53,7 @@
             "tag": "Tag",
             "status": "Status",
             "resourceTypes": "Resource types",
+            "dependsOn": "Subtending product blocks",
```

Comment:
I think it would be nice to have the english translation as close as possible to the key-name. When i openend the page i had to search a bit which column you added to the table. Therefore suggesting to rename `dependsOn` to `subtendingProductBlocks`

Reply:
check